src/arch/riscv/kernel/boot.S

```diff
@@ -74,6 +79,9 @@ _start:
 	j       kernel_start  # jump kernel
 .cfi_endproc
 
+die:
+	beq zero,zero, die
```

Comment:
```
die:
    b die
```

shouldn't it work?

Reply:
Or risc-v doesn't have unconditional jump?

Reply:
fixed it